### DISCUSSION TITLE
DCUNet performance

### DISCUSSION THREAD

**Discussion by ruizhecao96** [asteroid-team/asteroid#487] [General]
Did anyone try DCUNet on the Voice Bank-Demand dataset?  Since the original paper was retracted due to some mistakes they made, I am wondering about the true performance of DCUNet .

**Comment by mpariente** [comment]
@jonashaag might have answers on that.

**Comment by jonashaag** [comment]
There are other papers that report their own DCUNet performance, have a look here: https://scholar.google.com/scholar?cites=11525798829336957800&as_sdt=2005&sciodt=0,5&hl=de

Despite being an "older" model already I think it can still perform pretty well. My intuition is that for speech denoising the training data is now more important than the model. (Unless you care about computational efficiency, real-time use, etc., in which case you'd probably prefer something else.)

**Reply by ruizhecao96** [reply]
That's really helpful. Thank you very much!
